python/ray/tests/BUILD

```diff
@@ -426,6 +426,14 @@ py_test(
     deps = ["//:ray_lib", ":conftest"],
 )
 
+py_test(
+    name = "test_vsphere_node_provider",
+    size = "small",
+    srcs = ["gcp/test_vsphere_node_provider.py"],
```

Comment:
```suggestion
    srcs = ["vsphere/test_vsphere_node_provider.py"],
```

Reply:
I think this is causing the build failure

Reply:
Oh gosh, will fix this right now.

Reply:
Fxied in the latest commit, thanks for helping us do the troubleshooting!